domino_data/transfer.py

```diff
@@ -0,0 +1,82 @@
+from typing import BinaryIO, Generator, Optional
+
+import io
+import shutil
+import threading
+from concurrent.futures import ThreadPoolExecutor
+
+import urllib3
+
+MAX_WORKERS = 10
+MB = 2**20
```

Comment:
mmh maybe a bit more specific of a variable name here?

Reply:
that's a megabyte 2^20 bytes, I'll make it clearer